src/SprykerEco/Client/Unzer/UnzerDependencyProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Client\Unzer;
+
+use Spryker\Client\Kernel\AbstractDependencyProvider;
+use Spryker\Client\Kernel\Container;
+use SprykerEco\Client\Unzer\Dependency\Client\UnzerToZedRequestClientBridge;
+
+class UnzerDependencyProvider extends AbstractDependencyProvider
+{
+    /**
+     * @var string
+     */
+    public const CLIENT_ZED_REQUEST = 'CLIENT_ZED_REQUEST';
+
+    /**
+     * @param \Spryker\Client\Kernel\Container $container
+     *
+     * @return \Spryker\Client\Kernel\Container
+     */
+    public function provideServiceLayerDependencies(Container $container)
```

Comment:
```suggestion
    public function provideServiceLayerDependencies(Container $container): Container
```

Reply:
method declaration should be the same as in parent class